File: Armageddon/src/main/java/de/tomjuri/armageddon/util/Ref.java
package de.tomjuri.armageddon.util;

import net.minecraft.client.Minecraft;
import net.minecraft.client.entity.EntityPlayerSP;
import net.minecraft.world.World;

public class Ref {
    public static Minecraft mc() { return Minecraft.getMinecraft(); }
    public static EntityPlayerSP player() { return mc().thePlayer; }
    public static World world() { return mc().theWorld; }
}

File: src/main/kotlin/de/tomjuri/armageddon/util/Ref.kt
package de.tomjuri.armageddon.util

import de.tomjuri.armageddon.Armageddon
import net.minecraft.client.Minecraft

val mc
    get() = Minecraft.getMinecraft()
val player
    get() = mc.thePlayer
val world
    get() = mc.theWorld
val gameSettings
    get() = mc.gameSettings
val armageddon
    get() = Armageddon.instance
val macro
    get() = Armageddon.instance.macro
val routeManager
    get() = Armageddon.instance.routeManager
val failsafe
    get() = Armageddon.instance.failsafe
val config
    get() = Armageddon.instance.config
val tracker
    get() = Armageddon.instance.tracker